src/read/coff/relocation.rs

```diff
@@ -58,6 +58,13 @@ impl<'data, 'file, R: ReadRef<'data>> Iterator for CoffRelocationIterator<'data,
                     pe::IMAGE_REL_AMD64_SECREL7 => (RelocationKind::SectionOffset, 7, 0),
                     typ => (RelocationKind::Coff(typ), 0, 0),
                 },
+                pe::IMAGE_FILE_MACHINE_RV64 => match relocation.typ.get(LE) {
+                    pe::IMAGE_REL_RISCV_ABSOLUTE => (RelocationKind::Absolute, 64, 0),
+                    pe::IMAGE_REL_RISCV_HI20 => (RelocationKind::Relative, 20, 2),
+                    pe::IMAGE_REL_RISCV_LOW12I => (RelocationKind::Relative, 12, 2),
+                    pe::IMAGE_REL_RISCV_LOW12S => (RelocationKind::Relative, 12, -2),
```

Comment:
Not sure if these are the correct `size` and `addend` values, or where exactly to look to find them. Took a best guess